### PR TITLE
[DOCS] Adds instructions on model install in air-gapped env

### PR DESCRIPTION
**Pull request by szabosteve**
## Overview

This PR adds instructions on how to install models from Hugging Face in an air-gapped environment.

### Preview

[Air-gapped model install](https://eland_542.docs-preview.app.elstc.co/guide/en/elasticsearch/client/eland/master/machine-learning.html#ml-nlp-pytorch-air-gapped)

### CODE REVIEW

File: docs/guide/machine-learning.asciidoc [hunk 1]
```diff
@@ -118,6 +118,57 @@ docker run -it --rm elastic/eland \
...
 
+[discrete]
+[[ml-nlp-pytorch-air-gapped]]
+==== Install models in an air-gapped environment 
+
+You can install models in a restricted or closed network by pointing the 
+`eland_import_hub_model` script to local files. 
+
+To be able to complete the process below, make sure that 
+https://git-lfs.com/[Git Large File Storage] is installed in your system.
```
Comment (davidkyle) [inline docs/guide/machine-learning.asciidoc:129]:
```suggestion
For an offline install of a Hugging Face model the model first needs to be cloned locally,
Git and https://git-lfs.com/[Git Large File Storage] are required to be installed in your system.
```

File: docs/guide/machine-learning.asciidoc [hunk 2]
```diff
@@ -118,6 +118,57 @@ docker run -it --rm elastic/eland \
...
+3. Use the `eland_import_hub_model` script with the `--hub-model-id` set to the 
+directory of the cloned model to install it:
++
+--
+[source,bash]
+----
+eland_import_hub_model \
+      --url 'XXXX' \
+      --hub-model-id /PATH/TO/MODEL \
+      --task-type ner --insecure --es-model-id bert-base-ner
```
[the remark this answers is not in the document]
Comment (davidkyle) [reply docs/guide/machine-learning.asciidoc:156]: Once uploaded to Elasticsearch the model will have the Id specified by`--es-model-id`. If `--es-model-id` is not set the model Id is derived from the file path in `--hub-model-id` with spaces and path delimiters converted to double underscore `__`.

File: docs/guide/machine-learning.asciidoc [hunk 3]
```diff
@@ -118,6 +118,57 @@ docker run -it --rm elastic/eland \
...
+If you use the Docker image to run `eland_import_hub_model` you must bind mount 
+the model directory, so the container can read the files:
+
+[source,bash]
+----
+docker run --mount type=bind,source=/PATH/TO/MODELS,destination=/models,readonly -it --rm elastic/eland \
+    eland_import_hub_model \
+      --url 'XXXX' \
+      --hub-model-id /models/bert-base-NER \
+      --task-type ner --insecure --es-model-id bert-base-ner
```
Comment (davidkyle) [inline docs/guide/machine-learning.asciidoc:167]:
```suggestion
      --task-type ner \
      --es-username elastic --es-password XXX \
      --es-model-id bert-base-ner
```

Same as above, we shouldn't have --insecure in our examples

Comment (szabosteve) [reply docs/guide/machine-learning.asciidoc:167]: @davidkyle I copied these commands from [this comment](https://github.com/elastic/eland/issues/502#issuecomment-1335193128). Shouldn't it be updated accordingly?

**Review by davidkyle — APPROVED** [review]
LGTM

**Pull request merged**